sycl/include/CL/sycl/aliases.hpp

```diff
@@ -71,7 +71,7 @@ using half = cl::sycl::detail::half_impl::half;
 
 __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
-using byte = std::uint8_t;
+using byte __SYCL2020_DEPRECATED("use std::byte instead") = std::uint8_t;
```

Comment:
"std::byte" is C++17: are we requiring it for SYCL now?

Reply:
For SYCL2020 we do. The warning won't be shown for older editions.